draft-ietf-moq-warp.md

```diff
@@ -316,6 +318,12 @@ Table 3: Allowed packaging values
 |:================|:==========|:=================|
 | LOC             | "loc"     | See RFC XXXX     |
 
+### Is Live {#islive}
+Location: T    Required: Required  JSON Type: Boolean
+
+True if new Objects will be added to the track.
+False if no new Objects will be added to the track.
```

Comment:
```suggestion
False if no new Objects will be added to the track. This is typically sent when the publisher has ended the track.
```

Reply:
> This is typically sent when the publisher has ended the track.

This is sent under two "typical" conditions:
1. If the track is VOD 
2. If the track was live and has now ended.